general/administration/hardware-acceleration.md

```diff
@@ -47,17 +46,35 @@ or using `lshw`:
 lshw -C display
 ```
 
+## Tips on H.264 / AVC 10-bit videos
+
+The hardware decoding of H.264 10-bit aka High10 profile video is not supported by any Intel, AMD or NVIDIA GPU.
+
+Please consider upgrading these videos to HEVC 10-bit aka Main10 profile if you want to offload your CPU usage during transcoding.
+
+## Tips for Intel Gen9 and Gen11+ when using VAAPI or QSV on Linux
+
+The Intel [Guc/Huc firmware](https://wiki.archlinux.org/title/intel_graphics#Enable_GuC_/_HuC_firmware_loading) must be enabled for optional Low-Power encoding (pre-Gen11 only supports LP H.264).
+
```

Comment:
We should add some documentation or links on how to enable it on Debian/Ubuntu based distros. The Arch wiki isn't useful in this cases.